src/scss/partials/_skipTo.scss

```diff
@@ -0,0 +1,15 @@
+/**
+ * Skip To Styling
+ * =================================
+ * Styles relating to the "skip link" element.
+ */
+
+ .skipTo {
```

Comment:
@ashleynolan This class doesn't use the `c-` prefix, should it? I noticed that the language switcher also doesn't use the prefix.